### DISCUSSION TITLE
Installare HA su Odroid-XU4 con boot da SSD?

### DISCUSSION THREAD

**Discussion by roby70cr** [indomus/forum#2337] [GENERALE - Discussioni generiche]
Buongiorno,
vorrei usare un SSD per far partire ed usare HA o almeno per salvarne i dati, visto la precarietà delle SD e la difficoltà che ho ad istallare HA su eMMC e farlo girare.
E' possibile una configurazione del genere? Oppure è necessario sempre eseguire il boot da SD/eMMC e installare HA su SSD? Bisogna sempre installare un OS su SSD e quindi poi installare HA in contenitore?
Sono alquanto confuso.... chi può aiutarmi a chiarire come poter fare?
Grazie.

**Comment by snakuzzo** [comment]
Hai già provato a dare un'occhiata [QUI](https://www.home-assistant.io/installation/odroid/) e [QUI](https://community.home-assistant.io/t/hassos-boot-from-ssd-with-odroid-n2/315594/7)?

**Comment by roby70cr** [comment]
ciao snakuzzo, si, grazie, ho seguito la guide del primo link, ma senza successo. Se flasho l'immagine su eMMC il sistema non si avvia anche dopo parecchio tempo, l'unica cosa che sono riuscito a fare è flashare Dietpi e farlo avviare da eMMC, e da Dietpi installare la versione ottimizzata di HA, si avvia, ma il supervisore non parte per un errore di autenticazione (che non capisco quale sia, anche leggendo i log).
Porverò con la guida del secondo link, dovrebbe essere quello che fa al caso mio, anche se l'N2 presenta delle differenze con l'XU4.
Grazie intanto, mi metto all'opera sperando di avere hardware di supporto adeguato.
